packages/docs-site/src/components/presenters/Masthead/index.js

```diff
@@ -1,12 +1,15 @@
 import PropTypes from 'prop-types'
 import React, { useState } from 'react'
+import { Link } from 'gatsby'
 import {
   Button,
   Icons,
   Nav,
   PhaseBanner,
 } from '@royalnavy/react-component-library'
 
+// import { ArrowForward } from '@royalnavy/icon-library'
```

Comment:
Can we remove this line if it's no longer needed?

Reply:
Good spot. Sorry, this keeps happening.